docs/_optimist/storage/localstorage/index.de.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Localstorage
+lang: de
+permalink: /optimist/storage/localstorage/
+nav_order: 3200
+parent: Storage
+has_children: true
```

Comment:
Set to `false` if there are no children. Avoids having an empty TOC at the end of the page.

Reply:
done